docs/management/admin/response-actions.asciidoc

```diff
@@ -92,6 +92,18 @@ Required privilege: *Process Operations*
 
 Example: `suspend-process --pid 123 --comment "Suspend suspicious process"`
 
+=== `get-file`
+
+Retrieve a file from a host. You must include the following parameter to specify the file's location:
+
+* `--path` : The file's full path (including file name).
+
+Required privilege: *File Operations*
+
+Example: `get-file --path "/full/path/to/file.txt" --comment "Possible malware"`
+
+NOTE: When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a Malware Prevention Alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`) and paste it into the `--path` parameter.
```

Comment:
Looks great!  My one suggestion is to also say that the the Quarantined file path will be in the Highlighted fields of the alert details for convenience.

Ex:
<img width="893" alt="image" src="https://user-images.githubusercontent.com/56395104/224163405-e8832c8c-965f-44c2-8993-f6c795331911.png">

Reply:
Addressed in https://github.com/elastic/security-docs/pull/3043/commits/5f48b4ddc59112bbb4318533ace444128ac98598